.drone.yml

```diff
@@ -10,8 +10,10 @@ pipeline:
       - mkdir -p /go/bin
       - "curl -s https://glide.sh/get | sh"
       - glide install
+      - go get github.com/vektra/mockery/.../
```

Comment:
There is no easy way to lock down a particular version using `go get ...`. One way would be to check in generated mocks.